Cargo.toml

```diff
@@ -18,8 +18,10 @@ chrono = "0.4.23"
 tantivy = "0.20.1"
 itertools = "0.10.5"
 futures = "0.3.26"
+pythonize = "0.19.0"
+serde = "~1.0"
 serde_json = "1.0.91"
 
 [dependencies.pyo3]
-version = "0.18.0"
-features = ["chrono", "extension-module"]
+version = "0.19.1"
```

Comment:
This change needs to wait for #98 to land.

Reply:
It's in 👍🏼